homeassistant/config.py

```diff
@@ -659,7 +659,14 @@ def stringify_invalid(
     - Give a more user friendly output for unknown options
     - Give a more user friendly output for missing options
     """
-    message_prefix = f"Invalid config for '{domain}'"
+    if "." in domain:
+        integration_domain, _, platform_domain = domain.partition(".")
```

Comment:
Side note: It's a bit confusing that the parameter is called `domain` when it could be the platform path.

Reply:
yes, I agree. Would it be better if it was named `platform_path`, then if there's no `.`, it's an integration domain?

Reply:
I don't think so. I don't have a good suggestion right now.